src/network/send.ts

```diff
@@ -148,7 +149,7 @@ export async function sendMessage({
         sphinxLogger.info(`[Network] => isBotMsg`, logging.Network)
         // return // DO NOT FORWARD TO TRIBE, forwarded to bot instead?
       }
-      if (msg.sender.role === constants.chat_roles.owner && msg.type === 0) {
+      if (msg.type === 0) {
```

Comment:
I'm lost on why we removed this check?

Reply:
I removed it because I am already checking to ensure only the tribe owner can execute this block of code: https://github.com/stakwork/sphinx-relay/blob/ml-bot/src/network/send.ts#L133